openroad.bzl

```diff
@@ -592,7 +592,7 @@ def build_openroad(
     stage_args = init_stage_dict(all_stage_names, stage_args)
     stage_args["clock_period"] = SDC_FILE
     stage_args["synth_sdc"] = SDC_FILE
-    stage_args["synth"].append("VERILOG_FILES=" + " ".join(verilog_files))
+    stage_args["synth"].append("VERILOG_FILES=" + " ".join(map(lambda v: native.package_name() + "/" + v, verilog_files)))
```

Comment:
I believe this is too simple if we want to pass in labels that are outputs of other rules.